cmd/autoheal/alerts_worker.go

```diff
@@ -192,6 +195,13 @@ func (h *Healer) runRule(rule *autoheal.HealingRule, alert *alertmanager.Alert)
 		return nil
 	}
 
+	// Increment the metric of requested heales.
+	h.incrementRequestedActions(
+		actionType,
```

Comment:
Instead of explicitly assigning strings to specify the type you could use `reflect.TypeOf` to find the type and pass it, or just pass type name:

```go
h.incrementRequestedActions(
    reflect.TypeOf(*action).Name(),
    ...,
)
```

That way if in the future we add new types, or modify the names, this wouldn't need to change.